### ISSUE TITLE
events --since"Beginning of Unix Time doesn't work"

### ISSUE THREAD

**Issue by aanm** [docker/docker#11555]
```bash
[aanm@localhost ~]$ sudo docker events --since='1970-01-01' --until='2015-03-22 03:15'
[aanm@localhost ~]$ sudo docker events --since='1970-01-02' --until='2015-03-22 03:15'
2015-03-22T03:14:01.000000000Z 0b1fc3ce7efcac2984c00e3e822c19036bcf6d2f0c63d451246ab0cd2b198682: (from docker:10501-Docker-events-docs-not-accordingly-with-rfc3339) create
2015-03-22T03:14:01.000000000Z 0b1fc3ce7efcac2984c00e3e822c19036bcf6d2f0c63d451246ab0cd2b198682: (from docker:10501-Docker-events-docs-not-accordingly-with-rfc3339) start
2015-03-22T03:14:42.000000000Z 0b1fc3ce7efcac2984c00e3e822c19036bcf6d2f0c63d451246ab0cd2b198682: (from docker:10501-Docker-events-docs-not-accordingly-with-rfc3339) die
2015-03-22T03:14:52.000000000Z 0b1fc3ce7efcac2984c00e3e822c19036bcf6d2f0c63d451246ab0cd2b198682: (from docker:10501-Docker-events-docs-not-accordingly-with-rfc3339) destro
```

I believe the fault is this line, why is this particular line there? https://github.com/docker/docker/blob/master/events/events.go#L72

**Comment by agtorre** [comment]
#dibs taking a look

**Comment by aanm** [comment]
@agtorre already fixed at #11547
